### PR TITLE
fix:product tag add if do not exist

### PR DESCRIPTION
**Pull request by kapilpaul**

### CODE REVIEW

File: assets/js/dokan.js [hunk 1]
```diff
@@ -1502,6 +1502,7 @@ jQuery(function($) {
         // Ajax search products tags
         $("#product_tag_search").select2({
             allowClear: false,
+            tags: true,
```
Comment (alamgircsebd) [inline assets/js/dokan.js:1505]: No need any changes on this file

Comment (sabbir1991) [reply assets/js/dokan.js:1505]: Please change this code on file `product-editor.js` not `dokan.js`

File: assets/css/style.css [hunk 2]
```diff
@@ -3099,6 +3099,9 @@ div.media-sidebar a.edit-attachment {
 .product-edit-container .dokan-feat-image-upload .image-wrap:hover a.close {
   display: block;
 }
+.product-edit-container .dokan-feat-image-upload .container-image-and-badge {
```
Comment (sabbir1991) [inline assets/css/style.css:3102]: Also, change this code on src file not build file.

**Review by shemakeswebthings — COMMENTED** [review]
What is the file path to the .js file to apply these changes? I don't see an assets/src/js folder. Thank you.

**Comment by alamgircsebd** [comment]
> What is the file path to the .js file to apply these changes? I don't see an assets/src/js folder. Thank you.

Hello @shemakeswebthings this is source file, you need implement on dokan.js file which one locate in assets/js
let  me know if you have any problem implement it. I am here assist you. 

Thanks!

**Comment by shemakeswebthings** [comment]
Okay I got it. Thank you!

**Pull request merged**